libs/ledger-live-common/src/families/hedera/api/network.ts

```diff
@@ -55,10 +55,18 @@ export async function getAccountBalance(
 
 let _hederaClient: hedera.Client | null = null;
 
+let _hederaBalanceClient: hedera.Client | null = null;
+
 function getClient(): hedera.Client {
   _hederaClient ??= hedera.Client.forMainnet().setMaxNodesPerTransaction(1);
 
   //_hederaClient.setNetwork({ mainnet: "https://hedera.coin.ledger.com" });
 
   return _hederaClient;
 }
+
+function getBalanceClient(): hedera.Client {
+  _hederaBalanceClient ??= hedera.Client.forMainnet();
```

Comment:
First time i see this operation, can you enlighten me about what is it ?
Thank you :)

Reply:
If you mean the `??=`, to be honest I had just copy/pasted the existing Client declaration and reused it.

From some googling it seems to be the nullish coalescing assignment: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment

I'm not entirely sure the reason it was used here, but it seems that it would prevent reassignment of the client which could be helpful?